usaspending_api/transactions/delta_models/summary_state_view.py

```diff
@@ -87,17 +88,37 @@
                 0
             ) AS NUMERIC(23, 2)
         ) AS face_value_loan_guarantee,
-        COUNT(*) AS counts
+        COUNT(*) AS counts,
+        CAST(
+            COALESCE(
+                SUM(matching_awards.total_outlays),
+                0
+            ) AS NUMERIC(23, 2)
+        ) AS total_outlays
     FROM
         int.transaction_normalized
+    JOIN
+        rpt.award_search as2 ON (transaction_normalized.award_id = as2.award_id)
     LEFT OUTER JOIN
         int.transaction_fpds ON (transaction_normalized.id = transaction_fpds.transaction_id)
     LEFT OUTER JOIN
         int.transaction_fabs ON (transaction_normalized.id = transaction_fabs.transaction_id)
+    LEFT JOIN
+        (
+            SELECT
+                as2.award_id,
+                as2.total_outlays
+            FROM
+                rpt.award_search as2
```

Comment:
If this is referencing Award Search, then the `load_query_to_delta-summary_state_view` job in the pipeline needs to be moved to after Award Search is refreshed with `load_query_to_delta-award_search`. This will require a separate PR to the `usaspending-config` repo